app/models/audit_record.rb

```diff
@@ -1,23 +1,15 @@
 class AuditRecord < ApplicationRecord
   belongs_to :school
 
-  TYPE_DELETE_ACCOUNT = 'delete_account'
-  TYPE_ADD_SCHOOL_ADMIN = 'add_school_admin'
-  TYPE_REMOVE_SCHOOL_ADMIN = 'remove_school_admin'
-  TYPE_DROPOUT_STUDENT = 'dropout_student'
-  TYPE_MERGE_USER_ACCOUNTS = 'merge_user_accounts'
-  TYPE_UPDATE_EMAIL = 'update_email'
+  enum audit_type: {
+    delete_account: 'delete_account',
+    add_school_admin: 'add_school_admin',
+    remove_school_admin: 'remove_school_admin',
+    dropout_student: 'dropout_student',
+    merge_user_accounts: 'merge_user_accounts',
+    update_email: 'update_email',
+    update_name: 'update_name'
+  }
 
-  def self.valid_audit_types
-    [
-      TYPE_DELETE_ACCOUNT,
-      TYPE_ADD_SCHOOL_ADMIN,
-      TYPE_REMOVE_SCHOOL_ADMIN,
-      TYPE_DROPOUT_STUDENT,
-      TYPE_MERGE_USER_ACCOUNTS,
-      TYPE_UPDATE_EMAIL
-    ].freeze
-  end
-
-  validates :audit_type, presence: true, inclusion: { in: valid_audit_types }
+  validates :audit_type, presence: true, inclusion: { in: audit_types.values }
```

Comment:
This is redundant. As you have declared `audit_type` as an enum with a set of values, ActiveRecord will ensure that you are allowed to set only the valid keys.